drivers/espi/espi_saf_mchp_xec_v2.c

```diff
@@ -7,15 +7,15 @@
 
 #define DT_DRV_COMPAT microchip_xec_espi_saf_v2
 
-#include <kernel.h>
+#include <zephyr/kernel.h>
 #include <soc.h>
 #include <errno.h>
-#include <drivers/clock_control/mchp_xec_clock_control.h>
-#include <drivers/espi.h>
-#include <drivers/espi_saf.h>
-#include <drivers/interrupt_controller/intc_mchp_xec_ecia.h>
-#include <dt-bindings/interrupt-controller/mchp-xec-ecia.h>
-#include <logging/log.h>
+#include <zephyr/drivers/clock_control/mchp_xec_clock_control.h>
+#include <zephyr/drivers/espi.h>
+#include <zephyr/drivers/espi_saf.h>
+#include <zephyr/drivers/interrupt_controller/intc_mchp_xec_ecia.h>
+#include <zephyr/dt-bindings/interrupt-controller/mchp-xec-ecia.h>
+#include <zephyr/logging/log.h>
```

Comment:
Please squash these changes into the commit that introduces this file

Reply:
squashed